share/completions/ajv.fish

```diff
@@ -0,0 +1,41 @@
+set command ajv
```

Comment:
These should be local-scoped, or you'll overwrite someone's variable named "command".

Reply:
Tho tbh just remove this entirely and use `command -c ajv`.

It's not like you're gonna copy these into other files anyway.